lesson25/src/js/validation.js

```
@@ -0,0 +1,67 @@
+const showErrorMessage = (target,message) => {
+    target.classList.add("border-2","border-red-500");
+
+    const errorMessage = document.createElement("p");
+    errorMessage.classList.add("field-invalid","text-sm","my-2","text-rose-600");
+	errorMessage.textContent = message;
+
+	const error = target.nextElementSibling;
+	if(!error){
+		target.insertAdjacentElement("afterend",errorMessage);
+	}else{
+		if(!(error.classList.contains("field-invalid"))){
+			target.insertAdjacentElement("afterend",errorMessage);
+		}
+	}
+}
+
+const removeErrorMessage = (target) => {
+    target.classList.remove("border-red-500");
+
+    const error = target.nextElementSibling;
+    if(error && error.classList.contains("field-invalid")){
+        error.remove();
+    }
+}
```

The code to show and hide the error messages, 
I want to make it smart somehow because it is convoluted and hard to understand.